webpack/assets/javascripts/react_app/components/Bookmarks/components/SearchModal/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SearchModal from './SearchModal';
+import { useForemanModal } from '../../../ForemanModal/ForemanModalHooks';
+import { BOOKMARKS_MODAL } from '../../BookmarksConstants';
+import { translate as __ } from '../../../../common/I18n';
+import { noop } from '../../../../common/helpers';
+
+const ConnectedSearchModal = props => {
+  const { setModalClosed } = useForemanModal({
+    id: BOOKMARKS_MODAL,
+  });
+
+  return <SearchModal setModalClosed={setModalClosed} {...props} />;
```

Comment:
can we pass the `setModalClosed` action from the Bookmarks component as it exist there?
than we won't need this extra file and extra connection to redux.

Reply:
@laviro Thanks, updated!